_pkgdown.yml

```diff
@@ -40,6 +40,7 @@ reference:
     - parameters.workflow
     - tunable
     - tune_args
+    - message_wrap
```

Comment:
What is this?

Reply:
I wrote a function that uses `message()` but respects the current page width.

Reply:
https://github.com/tidymodels/tune/blob/246983accbe756c9044138096d99067fa20413f9/R/logging.R#L29

I had not listed it on the `pkgdown` site.